channels/uk.m3u

```diff
@@ -210,6 +222,16 @@ https://vs-hls-pushb-uk-live.akamaized.net/x=3/i=urn:bbc:pips:service:cbbc_hd/t=
 https://vs-hls-pushb-uk-live.akamaized.net/x=3/i=urn:bbc:pips:service:cbeebies_hd/mobile_wifi_main_sd_abr_v2_akamai_hls_live_http.m3u8
 #EXTINF:-1 tvg-id="CBeebiesHD.uk" tvg-name="CBeebies HD" tvg-country="UK" tvg-language="English" tvg-logo="" group-title="",CBeebies HD (720p) [Geo-Blocked]
 https://vs-hls-pushb-uk-live.akamaized.net/x=3/i=urn:bbc:pips:service:cbeebies_hd/t=3840/v=pv14/b=5070016/main.m3u8
+#EXTINF:-1 tvg-id="CBSDrama.uk" tvg-name="CBS Drama" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/fcsOntA.png" group-title="",CBS Drama
+http://stream.tvtap.live:8081/live/cbsdrama.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="CBSReality.uk" tvg-name="CBS Reality" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/k28RZK2.jpg" group-title="",CBS Reality
```

Comment:
Originates from the US.

Reply:
https://en.wikipedia.org/wiki/CBS_Reality

Reply:
CBS Reality is owned by CBS UK Division.

Reply:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.